assume/common/utils.py

```diff
@@ -116,23 +116,25 @@ def make_market_config(
     market_config = MarketConfig(
         name=id,
         market_products=market_products,
+        product_type=market_params.get("product_type", "energy"),
         opening_hours=rr.rrule(
             freq=freq,
             interval=interval,
             dtstart=start,
             until=end,
             cache=True,
         ),
-        additional_fields=market_params.get("additional_fields", []),
-        product_type=market_params.get("product_type", "energy"),
         opening_duration=pd.Timedelta(market_params["opening_duration"]),
+        market_mechanism=market_params["market_mechanism"],
+        maximum_bid_volume=market_params["maximum_bid_volume"],
+        maximum_bid_price=market_params["maximum_bid_price"],
+        minimum_bid_price=market_params["minimum_bid_price"],
```

Comment:
this should be `.get(...)` calls to be optional

Reply:
If I use .get and the value is not available, it return the value as None. And so since the value exists but is None, it overwrites the default values we defined in the class. Do you have any idea for a nicer solution to this instead of if else statements in the market class?